src/components/Dialogs/PartialDelegateDialog/PartialDelegationSuccess.tsx

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+import { Button } from "@/components/ui/button";
+import BlockScanUrls from "@/components/shared/BlockScanUrl";
+import Tenant from "@/lib/tenant/tenant";
+
+interface Props {
+  closeDialog: () => void;
+  hash: `0x${string}`;
+}
+
+export const PartialDelegationSuccess = ({ hash, closeDialog }: Props) => {
+  const { ui } = Tenant.current();
+  return (
+    <div>
+      <div className="w-full">
+        <Image
+          className="w-full"
+          src={ui.assets.success}
+          alt="Delegation successful image"
+        />
+      </div>
+
+      <h1 className="mt-4 mb-2 text-2xl font-extrabold text-primary">
+        You&apos;ve delegated your votes!
```

Comment:
I always forget &apos; so I've just been wrapping in `` like {`You've delegated your votes`} not sure if you'd like that pattern also!